service/raft/raft_address_map.hh

```diff
@@ -317,20 +334,22 @@ public:
         if (addr == gms::inet_address{}) {
```

Comment:
Regarding `opt_add_entry` - I wonder if perhaps we should give priority to it? After all those are the messages we receive from the node directly.

Reply:
opt_add_entry never resets an existing entry, regardless of generation. this is unchanged by this patch. This is correct, since opt_add_entry purpose is to speed up initial communication on shard 0 during initial bootstrap of group0, before gossiper has caught up. it serves no other purpose - later nodes always go through group0 first which holds the hosts table now.